### PR TITLE
Hyperparameter tuning scripts

### PR DESCRIPTION
**Pull request by ktang16**
- Update train.py with learning rate scheduler and change output for MacOS
- Add models.py for different models
- Add hyperparam_tune.py and hyperparam_tune.sh for hyperparameter tuning jobs on PACE

### CODE REVIEW

**Comment by o-castaneda** [comment]
This looks amazing, thanks for this. I only made a few modifications:

1. Added option to unfreeze layers in MyModel
2. Replace ksotang with $USER/$u in bash scripts so that any of us 3 could run the scripts on PACE

**Review by o-castaneda — APPROVED** [review]
